<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7391" started-at="2018-03-20T23:56:22Z" finished-at="2018-03-20T23:56:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7391" started-at="2018-03-20T23:56:22Z" finished-at="2018-03-20T23:56:30Z">
      <class name="test.mercado.libre.TestRecorrerSlide">
        <test-method status="PASS" signature="prepareClass()[pri:0, instance:test.mercado.libre.TestRecorrerSlide@4466af20]" name="prepareClass" is-config="true" duration-ms="2401" started-at="2018-03-20T23:56:22Z" finished-at="2018-03-20T23:56:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareClass -->
        <test-method status="PASS" signature="prepareTest()[pri:0, instance:test.mercado.libre.TestRecorrerSlide@4466af20]" name="prepareTest" is-config="true" duration-ms="919" started-at="2018-03-20T23:56:25Z" finished-at="2018-03-20T23:56:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prepareTest -->
        <test-method status="FAIL" signature="recorrerSlide()[pri:0, instance:test.mercado.libre.TestRecorrerSlide@4466af20]" name="recorrerSlide" duration-ms="3911" started-at="2018-03-20T23:56:25Z" finished-at="2018-03-20T23:56:29Z">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
at java.util.ArrayList.rangeCheck(ArrayList.java:657)
at java.util.ArrayList.get(ArrayList.java:433)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementListHandler.invoke(LocatingElementListHandler.java:39)
at com.sun.proxy.$Proxy25.get(Unknown Source)
at test.mercado.libre.MercadoLibreHomePage.irAlSiguiente(MercadoLibreHomePage.java:92)
at test.mercado.libre.TestRecorrerSlide.recorrerSlide(TestRecorrerSlide.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- recorrerSlide -->
        <test-method status="PASS" signature="cleanup()[pri:0, instance:test.mercado.libre.TestRecorrerSlide@4466af20]" name="cleanup" is-config="true" duration-ms="127" started-at="2018-03-20T23:56:29Z" finished-at="2018-03-20T23:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanup -->
      </class> <!-- test.mercado.libre.TestRecorrerSlide -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
